river/ensemble/streaming_random_patches.py

```diff
@@ -145,7 +161,7 @@ def __init__(
         self.metric = metric
         self.nominal_attributes = nominal_attributes if nominal_attributes else []
         self.seed = seed
-        self._rng = check_random_state(self.seed)
+        self._rng = np.random.default_rng(self.seed)
```

Comment:
I thought about that, however, it is still unclear the impact on other classes. The downside of replacing `RandomState` is that the results won't match with those from previous versions. In some cases, we better go with Python's rng. I am ok with creating an issue for this although it might not be a priority. Comments @smastelini @MaxHalford ?

Reply:
As I mentioned earlier, I would use Python's random function whenever possible. We can incrementally change unnecessary numpy usage in the long run.